Source/Core/VideoCommon/TextureConfig.h

```diff
@@ -31,14 +31,22 @@ struct TextureConfig
   u32 layers = 1;
   AbstractTextureFormat format = AbstractTextureFormat::RGBA8;
   bool rendertarget = false;
+};
+
+namespace std
+{
+template <>
+struct hash<TextureConfig>
+{
+  using argument_type = TextureConfig;
```

Comment:
Just curious, why do this?

Reply:
It's a requirement of `std::hash`

20.9.13

> The unordered associative containers defined in 23.5 use specializations of the class template hash as the default hash function. For all object types Key for which there exists a specialization hash<Key>, and for all enumeration types (7.2) Key, the instantiation hash<Key> shall:
>
> ...
>
> 1.3 - provide two nested types `result_type` and `argument_type` which shall be synonyms for size_t and Key, respectively